models/text-editor/src/migration.ts

```diff
@@ -145,9 +145,11 @@ async function processFixMigrateMarkupFor (
           const value = (doc as any)[attribute.name]
           if (value != null) {
             let res = value
-            while (true) {
+            while ((res as string).includes('\\"type\\"')) {
```

Comment:
I just realized this won't work for double escape 😀 
`"type"` -> `\"type\"` -> `\\\"type\\\"`

it should not be a problem, but we can check for `\"type\` (without last double quote)